c++/src/kj/compat/readiness-io.h

```diff
@@ -71,11 +71,21 @@ class ReadyOutputStreamWrapper {
   kj::Promise<void> whenReady();
   // Returns a promise that resolves when write() will return non-null.
 
+  _::Deferred<Function<void()>> disableEagerPumping();
```

Comment:
`Function` requires allocations and virtual calls, and `_::Deferred` is an internal type which makes it not great for interface use. Maybe better to introduce a new hand-written type here?

Also in the bikeshedding department, you could call this `cork()`, and the returned type could be `Cork`, after Linux's similarly-purposed `TCP_CORK`. (Or ignore me if you don't like that idea...)

Reply:
Yeah, seems wise. And `cork` is good enough for me 👍